subhub/api/webhooks/routes/firefox.py

```diff
@@ -11,24 +13,19 @@
 
 class FirefoxRoute(AbstractRoute):
     def route(self):
-        sqs_client = boto3.client("sqs", region_name=CFG.AWS_REGION)
         try:
-            queues = sqs_client.list_queues(
-                QueueNamePrefix="DevSub"
-            )  # we filter to narrow down the list
-            logger.info(f"queues {queues}")
-            queue_url = queues["QueueUrls"][0]
-            logger.info(f"queue url {queue_url}")
-            msg = sqs_client.send_message(QueueUrl=queue_url, MessageBody=self.payload)
-
-            if msg["ResponseMetadata"]["HTTPStatusCode"] == 200:
-                logger.info(f"message sent to Firefox queue: {msg}")
+            sns_client = boto3.client("sns", region_name=CFG.AWS_REGION)
+            response = sns_client.publish(
+                TopicArn=CFG.TOPIC_ARN_KEY,
+                Message=json.dumps({"default": json.dumps(self.payload)}),
+                MessageStructure="json",
+            )
+            logger.info(f"message {self.payload}")
```

Comment:
Should this log string be updated?

Reply:
Definitely and it should have been included in the last force push, I'll fix.